awscli/argprocess.py

```diff
@@ -278,16 +301,22 @@ def __call__(self, cli_argument, value, **kwargs):
             be raised.
 
         """
+        split_event = event_name.split('.')[1:]
+        service_name = split_event[0]
+        operation_name = split_event[1]
```

Comment:
We can move this into the else clause so it only gets executed when we need to parse as shorthand.